village/src/components/SmurfForm.js

```diff
@@ -14,12 +15,19 @@ class SmurfForm extends Component {
     event.preventDefault();
     // add code to create the smurf using the api
 
-    this.setState({
-      name: '',
-      age: '',
-      height: ''
-    });
-  }
+  const { name, age, height } = this.state
+      axios.post('http://localhost:3333/smurfs', { name, age, height })
+      .then(response =>
+        this.props.refreshSmurf()
```

Comment:
this is giving us a type error because refreshSmurf was not created inside app, but should fire the function effectively once refreshSmurf is created there.